public/css/home.css

```diff
@@ -0,0 +1,15 @@
+.posDEL {
```

Comment:
Was this all generated by mix? Or was it manually added? I'm pretty sure all of our CSS should be compiled down into app.css but I could be mistaken...